src/db/parity_db.rs

```diff
@@ -69,6 +75,13 @@ impl DbColumn {
                         compression,
                         ..Default::default()
                     },
+                    DbColumn::EthMappings => parity_db::ColumnOptions {
+                        preimage: false,
+                        // This is needed for key retrieval.
+                        btree_index: true,
```

Comment:
Why is this needed for key retrieval?

Reply:
I'm not 100% sure. Hash column indexes should work at the moment (I'm testing them). But what if we need to implement GC on `Hash` to message the `Cid` later (like Lotus is doing, see https://github.com/filecoin-project/lotus/blob/master/node/impl/full/txhashmanager.go#L122).

Reply:
Since we want iteration for later implementation of GC on Eth mappings, let's keep it set to `true`.

Reply:
Actually we don't need it for the GC. But we need it to retrieve data by CID when exposing a set of RPC endpoints.

Reply:
Hash indexes are enough for iterating on column values. But I'm unsure if we can retrieve the key for the value for the mapping `{Hash => message Cid}.`

Reply:
I was able to retrieve the transaction `Hash` from the mapping value (message `Cid` and timestamp pair).